attestation/Cargo.toml

```diff
@@ -8,7 +8,9 @@ edition = "2021"
 arrayvec = { version = "0.7.2", default-features = false }
 const-oid = { version = "0.9.0", features = ["db"] }
 der = { version = "0.6.0", features = ["derive", "oid"] }
+digest = {version = "0.10.3", default-features = false }
```

Comment:
Not related, but commit message "It allows for _carrying publish_ a TCB evidence as a certificate custom
extension." seems off.